src/modules/0.0.18/aws_memory_db/entity/subnet_group.ts

```diff
@@ -0,0 +1,23 @@
+import { Column, Entity, Generated } from 'typeorm';
+
+import { cloudId } from '../../../../services/cloud-id';
+
+@Entity()
+export class SubnetGroup {
+  @Column()
+  @Generated('increment')
+  id: number;
+
+  @Column({ unique: true, primary: true })
+  @cloudId
+  subnetGroupName: string;
```

Comment:
Oh, I see, you made this one the primary key. Why have the autoincrementing id then?

Reply:
I'm leaning towards having both, the string column as PK when possible and the auto-incremental "db id" in case someone wants to perform custom queries and do not want to use a string id, but a numeric one. We would still use the string internally as FK to improve UX.